converter_test.go

```diff
@@ -363,6 +363,30 @@ func TestStringToValue(t *testing.T) {
 	} else if ts.UnixNano() != 1549491451123456789 {
 		t.Errorf("expected unix timestamp: 1549491451123456789, got %v", ts.UnixNano())
 	}
+
+	rowType = &execResponseRowType{Type: "array", Fields: []fieldMetadata{{Type: "array", Fields: []fieldMetadata{{Type: "fixed"}}}}}
```

Comment:
Can we have a E2E test as well (in structured_types_read.go)? How does it relate to `TestArrayOfArrays`?

Reply:
Ok, added a case in TestArrayOfArrays for a 3-dimensional array. Currently native arrow fails it, I'm working on that as well